### ISSUE TITLE
doesn't show place correctly

### ISSUE THREAD

**Issue by mahdimotamedi** [eregnier/vue2-gmap-custom-marker#1]
Unfortunately component make true. but place always is center of map.
i think something is wrong with find coordinate to show marker

**Comment by mahdimotamedi** [comment]
i found problem
in example marker position is with lat and lng

but in the codes it's Latitude and Longitude

please modify that
i hope you the best...

**Comment by mahdimotamedi** [comment]
please resolve that if you could. tnx

**Comment by eregnier** [comment]
Hi @mahdimotamedi 
I am not sure to understand the issue you meet with this component. Could you rephrase your issue please , so I can try to fix it.
thank you.

**Comment by mahdimotamedi** [comment]
In the example (Readme.md)
You wrote lat and lng vars for set coordinate
but in the codes, it must be Latitude and Longitude to work

It just need to change name of vars in the example code . Readme.md

**Comment by eregnier** [comment]
I did it there 0a9f511b1ea4034dd1411e29f42a5c60e64a93b9. thank you.
